ansible/group_vars/all/defaults.yaml

```diff
@@ -47,5 +43,9 @@ kubernetes_bins:
 
 sysprep: true
 
+download_images: Null
+images_local_bundle_dir: ""
```

Comment:
It would be helpful to have this default to the value you provided in the overrides.yaml in the PR description.

This could potentially make the flow a little easier.

Reply:
You can't change the default. The targets your overriding don't exist in most cases.